foundation-extensions/mobile-core/signals/README.md

```diff
@@ -37,8 +37,38 @@ import com.adobe.marketing.mobile.*;
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
-​ Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using Cocoapods.
+{% tab title="iOS (AEP 3.x)" %}
+​Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using Cocoapods.
+
+Add following pods in your `Podfile`:
+
+```text
+pod 'AEPCore'
+pod 'AEPServices'
+pod 'AEPRulesEngine'
```

Comment:
These two dependencies are brought in by AEPCore, we can omit them in this example.